src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -64,6 +69,35 @@ public static List<(string, Lazy<ExternalXrefSpec>)> Load(string filePath)
             return result;
         }
 
+        private static void LoadZipFile(Dictionary<string, Lazy<ExternalXrefSpec>> result, SourceInfo<string> url)
+        {
+            using (var archive = ZipFile.OpenRead(url.Value))
```

Comment:
>Value [](start = 54, length = 5)

so the zip file cannot be an external resource?

Reply:
The zip file is used as external resource

Reply:
There is no real usage we have a remote url of zip file. Even we have it, think it should be handled by restore